File: debug_admin.py
import subprocess, json
# Check admin user in DB via the backend prisma
result = subprocess.run(
    ["npx.cmd", "prisma", "db", "execute", "--stdin"],
    input='SELECT id, email, role, "isActive" FROM public."User" WHERE email = \'admin@marshal.com\';',
    capture_output=True, text=True, timeout=10,
    cwd="backend"
)
print("STDOUT:", result.stdout[:500])
print("STDERR:", result.stderr[:500])

File: backend/debug_admin.js
const { PrismaClient } = require('@prisma/client');
const p = new PrismaClient();
p.user.findUnique({
  where: { email: 'admin@marshal.com' },
  select: { passwordHash: true }
}).then(u => {
  if (u) {
    console.log('FULL_HASH:' + u.passwordHash);
  } else {
    console.log('NOT FOUND');
  }
  p.$disconnect();
}).catch(e => { console.log('ERR', e.message); p.$disconnect(); });
